device.go

```diff
@@ -28,6 +28,7 @@ func IsHorizontal(orient DeviceOrientation) bool {
 type Device interface {
 	Orientation() DeviceOrientation
 	IsMobile() bool
+	IsBrowser() bool
```

Comment:
We should be careful with this in the future given that this technically is a breaking change, but we can probably ignore that in this case.